### DISCUSSION TITLE
live stream

### DISCUSSION THREAD

**Discussion by Jayravalcode** [open-mmlab/mmengine#883] [Q&A] [answered]
I am working on AI based live stream project using mmtracking but i found an error in mmengine like (UserWarning: `Visualizer` backend is not initialized because save_dir is None.) in /mmengine/visualization/visualizer.py:170. So how i can solve this problem for my project ?

**Comment by C1rN09** [comment, marked as answer]
This should only be a warning and it doesn't affect the training process. However, if you want to enable `Visualizer`, you can explicitly configure the `save_dir` field in your config file, or through command line options like `python tools/train.py your/config/file --cfg-options visualizer.save_dir='your/path/to/save'`
